Makefile

```diff
@@ -24,4 +21,10 @@ endif
 	 sdk/src/main/java/com/bugsnag/android/Notifier.java
 
 
-
+# Makes a release
+release:
+ifeq ($(VERSION),)g
+	@$(error VERSION is not defined. Run with `make VERSION=number release`)
+endif
+	make VERSION=$(VERSION) bump && git commit -am "v$(VERSION)" && git tag v$(VERSION) \
```

Comment:
I'd revise this to only `git add` the files which should be affected by bump, just in case there's other changes. Though in that case, it might be advisable to prevent a release if there's already changes present before making the bump.